template/service/init.sls

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+include:
+  - .running
```

Comment:
Again`init.sls` could include the actual startup SLS so it mirrors `clean.sls` which manages the dead SLS.

Reply:
Here I see `running.sls` mirroring `clean.sls`.